tobac/utils/mask.py

```diff
@@ -0,0 +1,408 @@
+        else).
+
+    Returns
+    -------
+    variable_cube_out : iris.cube.Cube
+        Masked cube for untracked volume.
+    """
+
+    from dask.array import ma
+    from copy import deepcopy
+
+    variable_cube_out = deepcopy(variable_cube)
+    variable_cube_out.data = ma.masked_where(
+        mask.core_data() > 0, variable_cube_out.core_data()
+    )
+    return variable_cube_out
+
+
+def mask_cube(cube_in, mask):
+    """Mask cube where mask is not zero.
+
+    Parameters
+    ----------
+    cube_in : iris.cube.Cube
+        Unmasked data cube.
+
+    mask : iris.cube.Cube
+        Mask to use for masking, >0 where cube is supposed to be masked.
+
+    Returns
+    -------
+    variable_cube_out : iris.cube.Cube
+        Masked cube.
+    """
+
+    from dask.array import ma
+    from copy import deepcopy
+
+    cube_out = deepcopy(cube_in)
+    cube_out.data = ma.masked_where(mask.core_data() != 0, cube_in.core_data())
+    return cube_out
+
+
+def mask_cell(mask, cell, track, masked=False):
+    """Create mask for specific cell.
+
+    Parameters
+    ----------
+    mask : iris.cube.Cube
+        Cube containing mask (int id for tracked volumes 0 everywhere
+        else).
+
+    cell : int
+        Integer id of cell to create masked cube for.
+
+    track : pandas.DataFrame
+        Output of the linking.
+
+    masked : bool, optional
+        Bool determining whether to mask the mask for the cell where
+        it is 0. Default is False.
+
+    Returns
+    -------
+    mask_i : numpy.ndarray
+        Mask for a specific cell.
+    """
+
+    feature_ids = track.loc[track["cell"] == cell, "feature"].values
+    mask_i = mask_features(mask, feature_ids, masked=masked)
+    return mask_i
+
+
+def mask_cell_surface(mask, cell, track, masked=False, z_coord="model_level_number"):
+    """Create surface projection of 3d-mask for individual cell by
+    collapsing one coordinate.
+
+    Parameters
+    ----------
+    mask : iris.cube.Cube
+        Cube containing mask (int id for tacked volumes, 0 everywhere
+        else).
+
+    cell : int
+        Integer id of cell to create masked cube for.
+
+    track : pandas.DataFrame
+        Output of the linking.
+
+    masked : bool, optional
+        Bool determining whether to mask the mask for the cell where
+        it is 0. Default is False.
+
+    z_coord : str, optional
+        Name of the coordinate to collapse. Default is 'model_level_number'.
+
+    Returns
+    -------
+    mask_i_surface : iris.cube.Cube
+        Collapsed Masked cube for the cell with the maximum value
+        along the collapsed coordinate.
+
+    """
+
+    feature_ids = track.loc[track["cell"] == cell, "feature"].values
+    mask_i_surface = mask_features_surface(
+        mask, feature_ids, masked=masked, z_coord=z_coord
+    )
+    return mask_i_surface
+
+
+def mask_cell_columns(mask, cell, track, masked=False, z_coord="model_level_number"):
```

Comment:
I would suggest deleting this function as well, since it depends on a function that is now deleted.

Reply:
Good point, @snilsn . I've deleted it in my latest commit.